packages/devtools_app/lib/src/ui/flutter/service_extension_widgets.dart

```diff
@@ -116,6 +116,9 @@ class _ServiceExtensionButtonGroupState
 
   Widget _buildExtension(ExtensionState extensionState) {
     final description = extensionState.description;
+    // TODO(kenz): replace this with `devToolsToggleButton` from common_widgets
```

Comment:
make this a bug or do it as part of this CL so it doesn't get forgotten.

Reply:
done https://github.com/flutter/devtools/issues/1728